lib/src/countly_flutter.dart

```diff
@@ -641,6 +676,9 @@ class Countly {
     return result;
   }
 
+  /// Set the limit for stored requests
```

Comment:
Imagine a tone shift in 'make cake' => 'this bakes a cake with strawberries and rainbows'
so instead of 'Set limit for stored requests' => 'Sets a limit to the the request queue. If more ...'

This applies to all comments